composer.json

```diff
@@ -21,11 +21,11 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
-        "friendsofsymfony/http-cache": "^2.15",
-        "symfony/framework-bundle": "^4.4.0 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4.0 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4.0 || ^5.0 || ^6.0"
+        "php": "^8.1",
+        "friendsofsymfony/http-cache": "^3.x-dev",
```

Comment:
can we do `^2.15||3.x-dev` instead? the jeanberu integration does not allow the (not yet released http-cache 3)

Reply:
thanks